### ISSUE TITLE
When I train the model with 4 GPUs, should I set the sync_bn to true?

### ISSUE THREAD

**Issue by rockywind** [TRAILab/CaDDN#71]
Thank you for you help.

`parser.add_argument('--sync_bn', action='store_true', default=False, help='whether to use sync bn')`

**Comment by codyreading** [comment]
Yup, I set it as True when doing multi GPU training

**Comment by rockywind** [comment]
Thank you for your help! 
But,when I can't set the sync_bn as True, when I doing multi GPU inference.

**Comment by codyreading** [comment]
Yes, this only affects training, as this calculates the Batch Norm mean and variance across the full batch (all GPUs) when training. Inference these parameters are fixed, so you don't need to set anything there.  Please see [here](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html)

**Comment by rockywind** [comment]
Thank you very much!
